enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/state/machines/CoreStateMachines.java

```diff
@@ -181,6 +200,15 @@ public void dispatch( ReplicatedLockTokenRequest lockRequest, long commandIndex,
         }
 
         @Override
+        public void dispatch( DummyRequest dummyRequest, long commandIndex, Consumer<Result> callback )
+        {
+            if ( benchmarkMachine != null )
+            {
+                benchmarkMachine.applyCommand( dummyRequest, commandIndex, callback );
+            }
```

Comment:
If `benchmarkMachine` should never be null then I'd get rid of the null check and let it bomb with an NPE if it's not set up correctly.